src/server/hotkey_collector.cpp

```diff
@@ -88,6 +107,7 @@ void hotkey_collector::on_start_detect(dsn::replication::detect_hotkey_response
         dwarn_replica(hint);
         return;
     case hotkey_collector_state::STOPPED:
+        _collector_start_time = dsn_now_s();
```

Comment:
Why update the start time in stopped state?

Reply:
because in these codes, the state will be transformed from `hotkey_collector_state::STOPPED` to `hotkey_collector_state::COARSE_DETECTING` which means the collecting action is beginning.